lib/process-manager.js

```diff
@@ -0,0 +1,29 @@
+/** @babel */
+
+import childProcess from 'child_process'
+import kill from 'tree-kill'
+
+export default class ProcessManager {
+  processes = new Set()
+
+  exec (command, options) {
```

Comment:
Rename this to something like `spawnChildProcess`, `executeChildProcess`, or `startChildProcess`?